install

```diff
@@ -350,13 +352,13 @@ sudo "/bin/mkdir", "-p", HOMEBREW_CACHE unless File.directory? HOMEBREW_CACHE
 sudo "/bin/chmod", "g+rwx", HOMEBREW_CACHE if chmod? HOMEBREW_CACHE
 sudo CHOWN, ENV["USER"], HOMEBREW_CACHE if chown? HOMEBREW_CACHE
 sudo CHGRP, group, HOMEBREW_CACHE if chgrp? HOMEBREW_CACHE
-system "/usr/bin/touch", "#{HOMEBREW_CACHE}/.cleaned" if File.directory? HOMEBREW_CACHE
+FileUtils.touch "#{HOMEBREW_CACHE}/.cleaned" if File.directory? HOMEBREW_CACHE
 
 if should_install_command_line_tools?
   ohai "Searching online for the Command Line Tools"
   # This temporary file prompts the 'softwareupdate' utility to list the Command Line Tools
   clt_placeholder = "/tmp/.com.apple.dt.CommandLineTools.installondemand.in-progress"
-  sudo "/usr/bin/touch", clt_placeholder
+  FileUtils.touch clt_placeholder
```

Comment:
Since `should_install_command_line_tools?` is `false` on Linux, there's no need to change this line.

Reply:
Yes, I know. But for clarity I wanted to avoid using two different methods for the same task, especially when the calls are only a few lines apart.

Reply:
A good intention, but the old call was `sudo "/usr/bin/touch"`, so the two are not equivalent.

Reply:
Ok, fixed.